images/gitness/config/template.apko.yaml

```diff
@@ -0,0 +1,36 @@
+contents:
+  packages:
+    - gitness
```

Comment:
```suggestion
```

If we don't drop it from here then adding `gitness-0.x` or whatever will conflict at build time.